<!--- If your issue is about a bug, please help us to reproduce and fix it by describing step by step in detail what you were doing when the error occurred. Feel free to delete irrelevant sections. Thanks. --->

## Description of your problem



## What steps will reproduce the problem?

1. 
2. 
3. 

## What is the expected output? What do you see instead?



### Paste traceback/error below (if applicable)
*(Copy from error dialog or `View > Panes > Internal Console`)*

```python-traceback

<!--- PASTE TRACEBACK HERE --->

```

## Key versions and other information:

* Spyder version:
* Version of spyder-unittest plugin:
* Installation method for Spyder and the unittest plugin: Anaconda / pip / ...
* Python version:
* Testing framework used: nose2 / pytest / unittest
* Testing framework version:
* Operating system:
